deeprankcore/dataset.py

```diff
@@ -405,30 +388,23 @@ def __init__( # pylint: disable=too-many-arguments, too-many-locals
         target_filter: Optional[Dict[str, str]] = None,
         check_integrity: bool = True
     ):
-        """
-        Class to load the .HDF5 files data into graphs.
+        """Class to load the .HDF5 files data into graphs.
 
         Args:
             hdf5_path (Union[str,list]): Path to .HDF5 file(s). For multiple .HDF5 files, insert the paths in a List. Defaults to None.
-
             subset (List[str], optional): List of keys from .HDF5 file to include. Defaults to None (meaning include all).
-
             target (str, optional): Default options are irmsd, lrmsd, fnat, binary, capri_class, dockq, and BA. It can also be a custom-defined target
                 given to the Query class as input (see: `deeprankcore.query`); in this case, the task parameter needs to be explicitly specified as well.
                 Only numerical target variables are supported, not categorical. If the latter is your case, please convert the categorical classes into
                 numerical class indices before defining the :class:`GraphDataset` instance. Defaults to None.
-
             task (str, optional): 'regress' for regression or 'classif' for classification. Required if target not in
                 ['irmsd', 'lrmsd', 'fnat', 'binary', 'capri_class', 'dockq', or 'BA'], otherwise this setting is ignored.
                 Automatically set to 'classif' if the target is 'binary' or 'capri_classes'.
                 Automatically set to 'regress' if the target is 'irmsd', 'lrmsd', 'fnat', 'dockq' or 'BA'.
-
```

Comment:
In the line below a square bracket is missing: node_features (Union[List[str], str`]`, optional). Same for edge_features below